<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Screen_Pacient_Data</class>
 <widget class="QWidget" name="Screen_Pacient_Data">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>801</width>
    <height>481</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="MyLabelAnimated" name="pb_aceptar">
   <property name="geometry">
    <rect>
     <x>540</x>
     <y>384</y>
     <width>160</width>
     <height>40</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>DIN Pro</family>
     <pointsize>15</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">QLabel {
     border: 3px solid #E3E3E3;
     border-radius: 5px;
     color: #E3E3E3
 }</string>
   </property>
   <property name="text">
    <string>ACEPTAR</string>
   </property>
   <property name="scaledContents">
    <bool>true</bool>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="MyLabelAnimated" name="pb_ajustar_fecha_hora">
   <property name="geometry">
    <rect>
     <x>180</x>
     <y>384</y>
     <width>190</width>
     <height>40</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>DIN Pro</family>
     <pointsize>15</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">QLabel {
     border: 3px solid #E3E3E3;
     border-radius: 5px;
     color: #E3E3E3
 }</string>
   </property>
   <property name="text">
    <string>FECHA / HORA</string>
   </property>
   <property name="scaledContents">
    <bool>true</bool>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="label_13">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>802</width>
     <height>482</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>MS Shell Dlg 2</family>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(17, 21, 25);</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="MyLineEdit" name="le_nombre">
   <property name="geometry">
    <rect>
     <x>267</x>
     <y>96</y>
     <width>434</width>
     <height>24</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>DIN Pro</family>
     <pointsize>14</pointsize>
     <weight>75</weight>
     <italic>false</italic>
     <bold>true</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">QLineEdit {
     border-radius: 3px;
     background-color: #E3E3E3;
     color: #444444
 }</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="MyLineEdit" name="le_CI_HC">
   <property name="geometry">
    <rect>
     <x>276</x>
     <y>133</y>
     <width>245</width>
     <height>24</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>DIN Pro</family>
     <pointsize>14</pointsize>
     <weight>75</weight>
     <italic>false</italic>
     <bold>true</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">QLineEdit {
     border-radius: 3px;
     background-color: #E3E3E3;
     color: #444444
 }</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="MyLineEdit" name="le_edad">
   <property name="geometry">
    <rect>
     <x>270</x>
     <y>170</y>
     <width>71</width>
     <height>24</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>DIN Pro</family>
     <pointsize>14</pointsize>
     <weight>75</weight>
     <italic>false</italic>
     <bold>true</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">QLineEdit {
     border-radius: 3px;
     background-color: #E3E3E3;
     color: #444444
 }</string>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="MyLineEdit" name="le_height">
   <property name="geometry">
    <rect>
     <x>440</x>
     <y>170</y>
     <width>51</width>
     <height>24</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>DIN Pro</family>
     <pointsize>14</pointsize>
     <weight>75</weight>
     <italic>false</italic>
     <bold>true</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">QLineEdit {
     border-radius: 3px;
     background-color: #E3E3E3;
     color: #444444
 }</string>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="MyLineEdit" name="le_weight">
   <property name="geometry">
    <rect>
     <x>620</x>
     <y>170</y>
     <width>61</width>
     <height>24</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>DIN Pro</family>
     <pointsize>14</pointsize>
     <weight>75</weight>
     <italic>false</italic>
     <bold>true</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">QLineEdit {
     border-radius: 3px;
     background-color: #E3E3E3;
     color: #444444
 }</string>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="label_9">
   <property name="geometry">
    <rect>
     <x>480</x>
     <y>170</y>
     <width>41</width>
     <height>24</height>
    </rect>
   </property>
   <property name="maximumSize">
    <size>
     <width>100</width>
     <height>16777215</height>
    </size>
   </property>
   <property name="font">
    <font>
     <family>DIN Pro</family>
     <pointsize>14</pointsize>
     <weight>75</weight>
     <italic>false</italic>
     <bold>true</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">QLabel {
     border-radius: 3px;
     background-color: #E3E3E3;
     color: #24292E
 }</string>
   </property>
   <property name="text">
    <string>cm</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="label_14">
   <property name="geometry">
    <rect>
     <x>670</x>
     <y>170</y>
     <width>31</width>
     <height>24</height>
    </rect>
   </property>
   <property name="maximumSize">
    <size>
     <width>100</width>
     <height>16777215</height>
    </size>
   </property>
   <property name="font">
    <font>
     <family>DIN Pro Cond Bold</family>
     <pointsize>14</pointsize>
     <weight>75</weight>
     <italic>false</italic>
     <bold>true</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">QLabel {
     border-radius: 3px;
     background-color: #E3E3E3;
     color: #24292E
 }</string>
   </property>
   <property name="text">
    <string>kg</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>100</x>
     <y>96</y>
     <width>60</width>
     <height>60</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="pixmap">
    <pixmap resource="assets.qrc">:/icons/paciente.png</pixmap>
   </property>
   <property name="scaledContents">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QLabel" name="l_icon_comentarios">
   <property name="geometry">
    <rect>
     <x>100</x>
     <y>216</y>
     <width>61</width>
     <height>120</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(17, 21, 25);</string>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="pixmap">
    <pixmap resource="assets.qrc">:/icons/comentarios.png</pixmap>
   </property>
   <property name="scaledContents">
    <bool>false</bool>
   </property>
   <property name="alignment">
    <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignTop</set>
   </property>
  </widget>
  <widget class="QLabel" name="label_11">
   <property name="geometry">
    <rect>
     <x>540</x>
     <y>133</y>
     <width>100</width>
     <height>24</height>
    </rect>
   </property>
   <property name="maximumSize">
    <size>
     <width>100</width>
     <height>16777215</height>
    </size>
   </property>
   <property name="font">
    <font>
     <family>DIN Pro</family>
     <pointsize>14</pointsize>
     <weight>75</weight>
     <italic>false</italic>
     <bold>true</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">QLabel {
     border-radius: 3px;
     background-color: #E3E3E3;
     color: #24292E
 }</string>
   </property>
   <property name="text">
    <string>Sexo: |</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="MyLineEdit" name="le_sexo">
   <property name="geometry">
    <rect>
     <x>620</x>
     <y>133</y>
     <width>81</width>
     <height>24</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>DIN Pro</family>
     <pointsize>14</pointsize>
     <weight>75</weight>
     <italic>false</italic>
     <bold>true</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">QLineEdit {
     border-radius: 3px;
     background-color: #E3E3E3;
     color: #444444
 }</string>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>180</x>
     <y>96</y>
     <width>91</width>
     <height>24</height>
    </rect>
   </property>
   <property name="maximumSize">
    <size>
     <width>100</width>
     <height>16777215</height>
    </size>
   </property>
   <property name="font">
    <font>
     <family>DIN Pro Cond Bold</family>
     <pointsize>14</pointsize>
     <weight>75</weight>
     <italic>false</italic>
     <bold>true</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">QLabel {
     border-radius: 3px;
     background-color: #E3E3E3;
     color: #24292E
 }</string>
   </property>
   <property name="text">
    <string>Nombre: |</string>
   </property>
   <property name="scaledContents">
    <bool>true</bool>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="label_6">
   <property name="geometry">
    <rect>
     <x>180</x>
     <y>170</y>
     <width>111</width>
     <height>24</height>
    </rect>
   </property>
   <property name="maximumSize">
    <size>
     <width>16777215</width>
     <height>16777215</height>
    </size>
   </property>
   <property name="font">
    <font>
     <family>DIN Pro</family>
     <pointsize>14</pointsize>
     <weight>75</weight>
     <italic>false</italic>
     <bold>true</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">QLabel {
     border-radius: 3px;
     background-color: #E3E3E3;
     color: #24292E
 }</string>
   </property>
   <property name="text">
    <string>  Edad: |</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="label_8">
   <property name="geometry">
    <rect>
     <x>360</x>
     <y>170</y>
     <width>91</width>
     <height>24</height>
    </rect>
   </property>
   <property name="maximumSize">
    <size>
     <width>100</width>
     <height>16777215</height>
    </size>
   </property>
   <property name="font">
    <font>
     <family>DIN Pro</family>
     <pointsize>14</pointsize>
     <weight>75</weight>
     <italic>false</italic>
     <bold>true</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">QLabel {
     border-radius: 3px;
     background-color: #E3E3E3;
     color: #24292E
 }</string>
   </property>
   <property name="text">
    <string>Talla: |</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="label_10">
   <property name="geometry">
    <rect>
     <x>540</x>
     <y>170</y>
     <width>100</width>
     <height>24</height>
    </rect>
   </property>
   <property name="maximumSize">
    <size>
     <width>100</width>
     <height>16777215</height>
    </size>
   </property>
   <property name="font">
    <font>
     <family>DIN Pro</family>
     <pointsize>14</pointsize>
     <weight>75</weight>
     <italic>false</italic>
     <bold>true</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">QLabel {
     border-radius: 3px;
     background-color: #E3E3E3;
     color: #24292E
 }</string>
   </property>
   <property name="text">
    <string>Peso: |</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="label_4">
   <property name="geometry">
    <rect>
     <x>180</x>
     <y>133</y>
     <width>100</width>
     <height>24</height>
    </rect>
   </property>
   <property name="maximumSize">
    <size>
     <width>100</width>
     <height>16777215</height>
    </size>
   </property>
   <property name="font">
    <font>
     <family>DIN Pro</family>
     <pointsize>14</pointsize>
     <weight>75</weight>
     <italic>false</italic>
     <bold>true</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">QLabel {
     border-radius: 3px;
     background-color: #E3E3E3;
     color: #24292E
 }</string>
   </property>
   <property name="text">
    <string> CI:/HC: |</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="MyPlainTextEdit" name="pt_comentarios">
   <property name="geometry">
    <rect>
     <x>180</x>
     <y>216</y>
     <width>521</width>
     <height>120</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>DIN Pro Cond Bold</family>
     <pointsize>14</pointsize>
     <weight>75</weight>
     <italic>false</italic>
     <bold>true</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">QPlainTextEdit {
     border-radius: 3px;
     background-color: #E3E3E3;
     color: #444444
 }</string>
   </property>
   <property name="plainText">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="l_version">
   <property name="geometry">
    <rect>
     <x>73</x>
     <y>0</y>
     <width>51</width>
     <height>32</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>DIN Pro</family>
     <pointsize>14</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(255, 255, 255);</string>
   </property>
   <property name="text">
    <string>...</string>
   </property>
   <property name="pixmap">
    <pixmap resource="assets.qrc">:/icons/version.png</pixmap>
   </property>
   <property name="scaledContents">
    <bool>false</bool>
   </property>
  </widget>
  <widget class="QLabel" name="label_5">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>71</width>
     <height>32</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="pixmap">
    <pixmap resource="assets.qrc">:/icons/version.png</pixmap>
   </property>
   <property name="scaledContents">
    <bool>false</bool>
   </property>
  </widget>
  <zorder>label_13</zorder>
  <zorder>pb_aceptar</zorder>
  <zorder>pb_ajustar_fecha_hora</zorder>
  <zorder>label_2</zorder>
  <zorder>le_nombre</zorder>
  <zorder>label_6</zorder>
  <zorder>label_8</zorder>
  <zorder>label_10</zorder>
  <zorder>label_4</zorder>
  <zorder>le_CI_HC</zorder>
  <zorder>label</zorder>
  <zorder>le_height</zorder>
  <zorder>label_9</zorder>
  <zorder>le_edad</zorder>
  <zorder>label_11</zorder>
  <zorder>le_sexo</zorder>
  <zorder>l_icon_comentarios</zorder>
  <zorder>le_weight</zorder>
  <zorder>label_14</zorder>
  <zorder>l_version</zorder>
  <zorder>label_5</zorder>
  <zorder>pt_comentarios</zorder>
 </widget>
 <customwidgets>
  <customwidget>
   <class>MyLabelAnimated</class>
   <extends>QLabel</extends>
   <header location="global">mylabelanimated.h</header>
  </customwidget>
  <customwidget>
   <class>MyLineEdit</class>
   <extends>QLineEdit</extends>
   <header location="global">mylineedit.h</header>
  </customwidget>
  <customwidget>
   <class>MyPlainTextEdit</class>
   <extends>QPlainTextEdit</extends>
   <header location="global">myplaintextedit.h</header>
  </customwidget>
 </customwidgets>
 <resources>
  <include location="assets.qrc"/>
 </resources>
 <connections/>
</ui>
